src/bin/tikv-server.rs

```diff
@@ -90,16 +90,19 @@ fn check_system_config(config: &TiKvConfig) {
     }
 
     for e in tikv_util::config::check_kernel() {
-        warn!("{:?}", e);
+        warn!("check-kernel";
+        "error" => %e);
     }
 
     // check rocksdb data dir
     if let Err(e) = tikv_util::config::check_data_dir(&config.storage.data_dir) {
-        warn!("rockdsb check data dir: {:?}", e);
+        warn!("rockdsb check data dir";
+              "error" => %e);
```

Comment:
rocksdb

Reply:
I'm not sure whether `rockdsb` is a keyword in some service which monitors the tikv's log

Reply:
Nop, but it might be something we need to monitor in future